pybloqs/block/table_formatters.py

```diff
@@ -53,12 +53,18 @@ class TableFormatter(object):
         Provides CSS styles to all <th> and <td> HTML tags.
     """
 
-    def __init__(self, rows=None, columns=None, apply_to_header_and_index=True):
-        """Initialise formatter and specify which rows and columns it is applied to. Default None applies to all."""
-        self.rows = rows
-        self.columns = columns
-        self.apply_to_header_and_index = apply_to_header_and_index
-        return
+    def __init__(self, rows=None, columns=None, apply_to_header_and_index=True, apply_to_header=False, apply_to_index=False):
+            """Initialise formatter and specify which rows and columns it is applied to. Default None applies to all."""
```

Comment:
This seems a little awkward. Needs a doc string explaining the behaviour.

`pbtf.TableFormatter(rows=None, columns=None, apply_to_header=False)` will have non intuitive behaviour

One alternative which would still be backward compatible might be to allow apply_to_header_and_index to accept a tuple.
e.g. (True, False) which refers to the header and index respectively.

Reply:
One tricky bit is the top left cell, as this would be affected by either apply_to_header and apply_to_index.

Reply:
@DominikMChrist @pablojim As suggested I have now allowed `apply_to_header_and_index` to accept a 2-tuple of Booleans in addition to the existing Boolean. The tuple refers to the header and index respectively, and is mapped to `apply_to_header` and `apply_to_index` internally (if a Boolean is supplied, it is supplied to both `apply_to_header` and `apply_to_index`). The tests have been (minimally) updated accordingly. Please let me know if you have any further comments.